test/e2e/framework/util.go

```diff
@@ -241,38 +248,44 @@ func GetServerArchitecture(c clientset.Interface) string {
 	return arch
 }
 
+// GetServicesProxyRequest returns a request for a service proxy.
 func GetServicesProxyRequest(c clientset.Interface, request *restclient.Request) (*restclient.Request, error) {
 	return request.Resource("services").SubResource("proxy"), nil
 }
 
-// unique identifier of the e2e run
-var RunId = uuid.NewUUID()
+// RunID is an unique identifier of the e2e run.
```

Comment:
s/an/a/ because of https://www.quora.com/Which-is-the-correct-grammar-usage-a-unique-or-an-unique

Reply:
done.

Reply:
It might be worth adding a warning: "Beware that this ID is not the same for all tests in the E2E run, because each  Ginkgo node creates it separately."

I prefer "E2E" over "e2e".

Reply:
Thank you for comment!! 😄 
I've added a warning comment, but I used "e2e" instead of "E2E".
Because "e2e" is used in [this doc](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) 🤔